libnetfilter_log.yaml

```
@@ -0,0 +1,62 @@
+package:
+  name: libnetfilter_log
+  version: 1.0.2
+  epoch: 0
+  description: programming interface (API) to the in-kernel connection tracking state table
+  copyright:
+    - license: GPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - libmnl-dev
+      - libnetfilter_conntrack-dev
+      - libnfnetlink-dev
+      - libtool
+      - pkgconf-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: git://git.netfilter.org/libnetfilter_log
+      tag: libnetfilter_log-${{package.version}}
+      expected-commit: 549db050ca94e44529435b377c08feaa833f917c
+
+  - runs: |
+      ./autogen.sh
+
+  - uses: autoconf/configure
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+subpackages:
+  - name: libnetfilter_log-dev
+    description: libnetfilter_log development files
+    pipeline:
+      - uses: split/dev
+    test:
+      environment:
+        contents:
+          packages:
+            - libmnl-dev
+            - libnfnetlink-dev
+      pipeline:
+        - uses: test/pkgconf
+        - uses: test/tw/ldd-check
```

```suggestion
        - uses: test/tw/ldd-check
        - uses: test/tw/header-check
```
if dev headers are required.